vnext/Scripts/Tfs/Layout-Headers.ps1

```diff
@@ -5,14 +5,26 @@ param(
 	[string] $SourceRoot = ($PSScriptRoot | Split-Path | Split-Path | Split-Path),
 	[string] $TargetRoot = "$SourceRoot\vnext\target",
 	[System.IO.DirectoryInfo] $ReactWindowsRoot = "$SourceRoot\vnext",
-	[System.IO.DirectoryInfo] $ReactNativeRoot = "$SourceRoot\vnext\build\" + @(gci "$ReactWindowsRoot\build\" react-native-patched -Recurse -Directory -Name)[0],
+	[System.IO.DirectoryInfo] $ReactNativeRoot,
 	[string] $FollyVersion = '2019.09.30.00',
 	[System.IO.DirectoryInfo] $FollyRoot = "$SourceRoot\node_modules\.folly\folly-${FollyVersion}",
 	[string[]] $Extensions = ('h', 'hpp', 'def')
 )
 
+if (!$ReactNativeRoot) {
+    $intermediateBuildDir =  if ($env:BaseIntDir) { $env:BaseIntDir } else { "$SourceRoot\vnext\build\" }
```

Comment:
Please, don't directly use environment variables.
Add another parameter which can be defaulted to the env. variable if needed.

This allows for local testing and custom workflows.

Reply:
Do you mean $ReactNativeRoot? $ReactNativeRoot isn't an environment variable. It's a command parameter (PowerShell prefixes variables with "$" in general)

Reply:
You can currently do `powershell Scripts/Tfs/LayoutHeaders.ps1 -ReactNativeRoot D:\foo` and all works correctly.

Reply:
I mean `$env:BaseIntDir`.
The script should not assume any environment variable.
There should be a separate parameter behind it, and then the environment variable could be the default value.

Reply:
> You can currently do `powershell Scripts/Tfs/LayoutHeaders.ps1 -ReactNativeRoot D:\foo` and all works correctly.

Ah, I see. `BaseIntDir` as an environment variable is a special, "optional" case.

It's alright, then.

Reply:
@JunielKatarn it looks like you're still marked as waiting for changes in the review. I believe this will block merging even if this otherwise gets sign ff.

Reply:
:shipit: